app/controllers/api/v1/users_controller.rb

```diff
@@ -32,6 +32,6 @@ def destroy
   private
   
   def user_params
-    params.require(:user).permit(:name, :email, :role, :language_pref, :movie_history, :password_digest) 
+    params.require(:user).permit(:name, :email, :role, :movie_history, :password, :password_confirmation) 
```

Comment:
Thanks for removing the `:language_pref` and it makes sense that we needed to split `:password_digest` into `:password` and `:password_confirmation`.